src/openforms/registrations/contrib/objects_api/tests/test_templatetags.py

```diff
@@ -0,0 +1,68 @@
+from unittest import TestCase
+
+from django.template import Context, Template
+
+from openforms.registrations.constants import RegistrationAttribute
+from openforms.submissions.tests.factories import SubmissionFactory
+
+
+class JsonSummeryTests(TestCase):
+    def test_render_json_summary(self):
+        submission = SubmissionFactory.from_components(
+            [
+                {
+                    "key": "voornaam",
+                    "type": "textfield",
+                    "registration": {
+                        "attribute": RegistrationAttribute.initiator_voornamen,
+                    },
+                },
+                {
+                    "key": "achternaam",
+                    "type": "textfield",
+                    "registration": {
+                        "attribute": RegistrationAttribute.initiator_geslachtsnaam,
+                    },
+                },
+                {
+                    "key": "tussenvoegsel",
+                    "type": "textfield",
+                    "registration": {
+                        "attribute": RegistrationAttribute.initiator_tussenvoegsel,
+                    },
+                },
+            ],
+            submitted_data={
+                "voornaam": "Foo",
+                "achternaam": "Bar",
+                "tussenvoegsel": "de",
+            },
+            language_code="en",
+        )
+        submission_step = submission.steps[0]
+        step_slug = submission_step.form_step.form_definition.slug
+
+        # Loads template tag and uses template tag
+        template = Template(
+            """{% load registrations.contrib.objects_api.json_summary %}{% json_summary %}"""
+        )
+
+        rendered = template.render(Context({"_submission": submission}))
```

Comment:
We have `openforms.template.render_from_string` for rendering templates from strings, see the documentation: https://open-forms.readthedocs.io/en/latest/developers/backend/core/templating.html

Using `django.template.Template` directly is a backwards compatibility thing from Django, but it can't handle (possible future) other template engines. It's best avoided.